### PR TITLE
Fantasy Premier League (Independent Publisher)

### PR DESCRIPTION
**Pull request by Flow-Joe**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [N/A] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

**Screenshots:**

Power Automate calling the connector and pushing to Microsoft Teams:
image

Actions running in a flow:
image

Output of actions:
image

image

image

image

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Thank you for your submission, @Flow-Joe!

File: independent-publisher-connectors/Fantasy Premier League/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2184 @@
...
+                            "description": "total"
+                          },
+                          "entry": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "entry"
+                          },
+                          "entry_name": {
+                            "type": "string",
+                            "description": "entry_name"
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Fantasy Premier League/apiDefinition.swagger.json:2134]: @Flow-Joe please make sure that descriptions look like valid sentences. You can refer to [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters) for definition examples and to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) for more information about strings.

File: independent-publisher-connectors/Fantasy Premier League/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2184 @@
...
+                      },
+                      "ict_index_rank_type": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "ict_index_rank_type"
+                      },
+                      "corners_and_indirect_freekicks_order": {
+                        "type": "integer",
+                        "description": "corners_and_indirect_freekicks_order",
+                        "title": "",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Fantasy Premier League/apiDefinition.swagger.json:1869]: @Flow-Joe please fill in or remove this property if it's not required.

File: independent-publisher-connectors/Fantasy Premier League/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Fantasy Premier League Independant Connector
+The Fantasy Premier League independant connector allows you to gather information from your fantasy football(soccer) leagues, ratings, information on current fixtures, general information and more. This will allow you to gather information on your fantasy league and post information about your league, such as current standings, to your Microsoft Teams groupchat or channel.
+
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Fantasy Premier League/README.md:3]: @Flow-Joe please add Publisher heading in readme.md file, it should look like `Publisher: Publisher's name`. You can refer to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md?plain=1) for more information about required headings.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hello @Flow-Joe,

Please check comments after the review. There're a few minor things that can be fixed.

Let us know if you have any questions.

**Comment by v-EgorKozhadei** [comment]
[[certify-connector]]

**Comment by v-EgorKozhadei** [comment]
Hello @Flow-Joe, 

I hope you are doing well. 

Congratulations, your connector is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.  

Please let us know if you have any questions. 

Thank you very much for working with us.

**Comment by Amjed-Ayoub** [comment]
Hello @Flow-Joe,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@Flow-Joe Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, thanks.

**Pull request merged**